ui-tests/src/test/java/io/syndesis/qe/pages/app/AppPage.java

```diff
@@ -0,0 +1,108 @@
+package io.syndesis.qe.pages.app;
+
+import static com.codeborne.selenide.Selenide.$;
+
+import org.apache.log4j.Logger;
+
+import org.openqa.selenium.By;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+
+import java.util.Map;
+
+/**
+ * Created by sveres on 11/8/17.
+ */
+public class AppPage {
+
+	private static final Logger log = Logger.getLogger(AppPage.class);
+
+	private static String baseurl = "/";
+
+	private SelenideElement rootElement = $(By.cssSelector("syndesis-root"));
+
+	public SelenideElement getButton(String buttonTitle) {
+		log.info(String.format("searching for button %s", buttonTitle));
+		return this.rootElement.find(By.xpath(String.format("//button[contains(text(), '%s')]", buttonTitle)));
+	}
+
+	//IS THIS METHOD NECESSARY? - NO, to be deleted.
+	public void clickOnFirstVisibleButton(String buttonTitle) {
```

Comment:
I'm not sure. This method is clearly different from the next one (clickButton()) even with shouldBe() on button element.

Reply:
ok. I got it.